project/interpolation_test.go

```diff
@@ -26,11 +27,20 @@ func testInvalidInterpolatedLine(t *testing.T, line string) {
 
 func TestParseLine(t *testing.T) {
 	variables := map[string]string{
-		"A": "ABC",
-		"X": "XYZ",
-		"E": "",
+		"A":           "ABC",
+		"X":           "XYZ",
+		"E":           "",
+		"lower":       "WORKED",
+		"MiXeD":       "WORKED",
+		"split_VaLue": "WORKED",
+		"9aNumber":    "WORKED",
 	}
 
+	testInterpolatedLine(t, "WORKED", "$lower", variables)
+	testInterpolatedLine(t, "WORKED", "${MiXeD}", variables)
+	testInterpolatedLine(t, "WORKED", "${split_VaLue}", variables)
+	testInterpolatedLine(t, "WORKED", "$9aNumber", variables)
```

Comment:
Okay, I can make thing starting with a number not work, but an embedded number would be fine?